content/scss/_s-colors.scss

```diff
@@ -32,4 +32,25 @@ $color-red-200: #FFB0B9;
 
 $color-green-500: #00A57D;
 $color-blue-500: #1267FB;
-$color-yellow-500: #F4C95D;
+$color-yellow-500: #F0F096;
+$color-orange-500: #FF8700;
+$color-brown-500: #A29490;
+$color-mauve-500: #744DFF;
+$color-mint-500: #8AFFC2;
+$color-beige-500: #FFFAE7;
+
+/* Semantic colors
+ ========================================================================== */
+
+$color-primary: #FF4054;
+$color-header: #000000;
+$color-text: #4D4D4D;
+$color-muted: #666666;
```

Comment:
Should we have some prose to explain what is the recommended usage of some of these ? (If yes, you can just create an issue, don't bother updating this PR.)